pkg/kubectl/cmd/patch.go

```diff
@@ -36,13 +37,16 @@ Please refer to the models in https://htmlpreview.github.io/?https://github.com/
 // Partially update a node using strategic merge patch
 kubectl patch node k8s-node-1 -p '{"spec":{"unschedulable":true}}'
 
+// Partially update a node from node.json using strategic merge patch
```

Comment:
@kargakis  The issue "Comments in kubectl examples should use bash comments, not Go comments #12426" was present, so I think we should use # instead of //.